docs/.vuepress/theme/components/NavLinks.vue

```diff
@@ -35,35 +135,222 @@ export default {
     }
   },
   methods: {
-    isActive
+    isActive,
+    handleSearchToggle(value) {
```

Comment:
Similar to `handleNavToggle`, I think this could be simplified to just toggle a boolean without passing a value.

Reply:
I'd prefer to keep this, it allows us to clear the search in certain scenarios i.e. when a link is clicked. imo it's much better to be explicit when there's a desired behaviour

Reply:
I don't follow - `it allows us to clear the search in certain scenarios i.e. when a link is clicked` - where are we doing this? From what I understand this merely sets `isSearchVisible` to true or false.

Reply:
Switched this out for the toggle like you mentioned — I honestly prefer being super explicit but I'll go with your judgement